### PR TITLE
Error term correction

### PR DESCRIPTION
**Pull request by bobbleoxs**
Please check course material under 23. Gradient Descent. 

Gradient Descent Step stated clearly for sigmoid binary cross-entropy error (which suits the student admission case) the GD step should be: 

w + a * (y - output) * x

There is no sigmoid prime required again.

### CODE REVIEW

**Comment by bobbleoxs** [comment]
Just noticed the 1-layer to 2-layer confusion. Understood. Close PR.
